app/google-services.json

```diff
@@ -0,0 +1,98 @@
+{
+  "project_info": {
+    "project_number": "977629908956",
+    "firebase_url": "https://sdp-firebase-bootcamp-30644-default-rtdb.europe-west1.firebasedatabase.app",
+    "project_id": "sdp-firebase-bootcamp-30644",
+    "storage_bucket": "sdp-firebase-bootcamp-30644.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:977629908956:android:23b7f01584c9d4fb330919",
+        "android_client_info": {
+          "package_name": "com.example.fireapp"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "977629908956-sigdt65kvtdcun6duqcoq3nkb146i0t5.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyB-85DOUKIhk0lGZ2Is8yfiIVbBiYX8qBc"

```

Comment:
Should the key be in the blank?

Reply:
This file was taken from the (BC2)-Firebase branch, but from my understanding it has been downloaded directly from the Firebase project. Therefore I prefer not modifying this file.